tools/data_manipulation/xarray/xarray_mapplot.xml

```diff
@@ -0,0 +1,346 @@
+            <option value="RdBu_r">RdBu_r</option>
+            <option value="RdGy_r">RdGy_r</option>
+            <option value="RdPu_r">RdPu_r</option>
+            <option value="RdYlBu">RdYlBu</option>
+            <option value="RdYlGn">RdYlGn</option>
+            <option value="Reds">Reds</option>
+            <option value="Set1">Set1</option>
+            <option value="Set2">Set2</option>
+            <option value="Set3">Set3</option>
+            <option value="Spectral">Spectral</option>
+            <option value="Wistia">Wistia</option>
+            <option value="YlGn">YlGn</option>
+            <option value="YlGnBu">YlGnBu</option>
+            <option value="YlOrBr">YlOrBr</option>
+            <option value="YlOrRd">YlOrRd</option>
+            <option value="afmhot">afmhot</option>
+            <option value="autumn">autumn</option>
+            <option value="binary">binary</option>
+            <option value="bone">bone</option>
+            <option value="brg">brg</option>
+            <option value="bwr">bwr</option>
+            <option value="cool">cool</option>
+            <option value="coolwarm" selected="true">coolwarm</option>
+            <option value="copper">copper</option>
+            <option value="cubehelix">cubehelix</option>
+            <option value="flag">flag</option>
+            <option value="gist_earth">gist_earth</option>
+            <option value="gist_gray">gist_gray</option>
+            <option value="gist_heat">gist_heat</option>
+            <option value="gist_ncar">gist_ncar</option>
+            <option value="gist_rainbow">gist_rainbow</option>
+            <option value="gist_stern">gist_stern</option>
+            <option value="gist_yarg">gist_yarg</option>
+            <option value="gnuplot">gnuplot</option>
+            <option value="gnuplot2">gnuplot2</option>
+            <option value="gray">gray</option>
+            <option value="hot">hot</option>
+            <option value="hsv">hsv</option>
+            <option value="jet">jet</option>
+            <option value="nipy_spectral">nipy_spectral</option>
+            <option value="ocean">ocean</option>
+            <option value="pink">pink</option>
+            <option value="prism">prism</option>
+            <option value="rainbow">rainbow</option>
+            <option value="seismic">seismic</option>
+            <option value="spring">spring</option>
+            <option value="summer">summer</option>
+            <option value="tab10">tab10</option>
+            <option value="tab20">tab20</option>
+            <option value="tab20b">tab20b</option>
+            <option value="tab20c">tab20c</option>
+            <option value="terrain">terrain</option>
+            <option value="winter">winter</option>
+        </param>
+        <param name="proj" type="text" optional="true" label='Specify the projection (proj4) on which we draw e.g. {"proj":"PlateCarree"} with double quote (optional)'>
```

Comment:
why not only ask to type "PlateCarree"?

Reply:
Sorry for my naive question.

Reply:
because in the Nordics, we use more complex projection and we need to specify addition parameters. for instance:
`{"proj":"EquidistantConic", "central_longitude": 20.0, "central_latitude": 70.0 }`

So I chose to use the standard proj4 "way" to specify projection.

Reply:
I see, maybe that example should be added to the help?